### PR TITLE
Shiny tutorial

### PR DESCRIPTION
**Pull request by bl2791**
This is a concise tutorial of shiny.

### CODE REVIEW

**Comment by skyetim** [comment]
So I should merge this instead of #33 right?

**Comment by bl2791** [comment]
Yes!

**Review by bojum — CHANGES_REQUESTED** [review]
1. There are 2 .rmd files. Please remove one of them. see [here](https://jtr13.github.io/cc19/index.html#preparing-your-.rmd-file).
2. In whichever .rmd file you keep, leave the second line blank. Your names should start on the third line.
3. I assume `resources/shiny_tutorial.zip` the resources for your project. Please create a new folder under`resources` for that. See [1.3.6](https://jtr13.github.io/cc19/index.html#submission-steps). Make sure your resource files are in a format that can be accessed by your .Rmd.

File: a_tutorial_of_shiny.Rmd [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+1. Go to https://www.shinyapps.io to sign up for an account. 
+
+![](resources/shiny_tutorial/shinyapps.io.png)
+
+2. When you signed up for a new account on https://www.shinyapps.io, there will be instructions on how to associate your account with your RStudio IDE and how to deploy your app.
+
+![](resources/shiny_tutorial/newuserinstructions.png) 
+  
+# Part 2 How to Customize Reactions
```
Comment (bojum) [inline a_tutorial_of_shiny.Rmd]: don't use a single hashtag header

File: a_tutorial_of_shiny.Rmd [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+
+In this tutorial, we will use the cars dataset as an example to wall through the process of building a shiny app. The cars dataset contains various information about a particular car.
+
+```{r, eval=FALSE}
+cars_info <- read.csv("cars.csv")
+```
+
+![](resources/shiny_tutorial/carsdata.png)
+
+# Part 1 How to Build a Shiny App
```
Comment (bojum) [inline a_tutorial_of_shiny.Rmd]: don't use a single hashtag header

**Review by bojum — CHANGES_REQUESTED** [review]
Just a couple more minor changes
- Please make sure that the first line of your .rmd file is the only line with a single hashtag header
- Please also remove shiny_tutorial.zip from resources/

File: a_tutorial_of_shiny.Rmd (hunk 2 above)
Comment (bl2791) [reply a_tutorial_of_shiny.Rmd]: I made a pull request again but I don't know why it failed.

**Comment by bojum** [comment]
Don't worry about this issue. Professor Robbins mentioned that 'After submitting your pull request, do not be concerned if you see an "All builds have failed" message from Travis CI. There are things that need to be done on the backend, such as adding the libraries you use to the project for the Travis CI build to pass.' We'll get back to you if anything needs to be changed.

**Comment by bojum** [comment]
In the meantime, would you mind trying removing the word 'setup' from your first R code chunk and committing again? 

...
{r **setup**, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
...

My guess is that there is already an r chunk with this 'setup' name in master branch and no repetition is allowed.

**Comment by bojum** [comment]
Sorry for the delay. There was a mistake on my side. Thank you for your contribution! 🎉

**Pull request merged**